fbpic/particles/particles.py

```diff
@@ -222,6 +222,10 @@ def dens_func( z, r ) ...
             self.prefix_sum_shift = 0
             # Register boolean that records if the particles are sorted or not
             self.sorted = False
+            # Define CUDA threads per block for deposition and gathering kernels
+            deposit_tpb = 16 if cuda_gpu_model == "V100" else 8
+            self.deposit_tpb = 32 if particle_shape == "cubic" else deposit_tpb
+            self.gather_tpb = 256 if particle_shape == "cubic" else 128
```

Comment:
While I like the concise one-line syntax `if .. else` for the other cases, here specifically I think that it would be clearer to do:
```
if particle_shape == "cubic":
    self.deposit_tpb = 32
    self.gather_tpb = 256
else:
    self.deposit_tpb = 16 if cuda_gpu_model == "V100" else 8
    self.gather_tpb = 128
```